### PR TITLE
systemd updates

### PR DESCRIPTION
**Pull request by jney**

### CODE REVIEW

**Comment by kr** [comment]
I was under the impression that port activation would take care of startup, so the WantedBy and After lines are not what we want. Was I wrong?

File: adm/systemd/beanstalkd.service [hunk 1]
```diff
@@ -3,4 +3,4 @@ Description=Beanstalkd is a simple, fast work queue
 
 [Service]
 User=nobody
-ExecStart=/usr/bin/beanstalkd
+ExecStart=/usr/local/bin/beanstalkd
```
Comment (tlstid) [inline adm/systemd/beanstalkd.service]: 5029159663

File: adm/systemd/beanstalkd.service [hunk 2]
```diff
@@ -1,6 +1,10 @@
 [Unit]
 Description=Beanstalkd is a simple, fast work queue
+After=network.target
 
 [Service]
 User=nobody
-ExecStart=/usr/bin/beanstalkd
+ExecStart=/usr/local/bin/beanstalkd
+
+[Install]
```
Comment (tlstid) [inline adm/systemd/beanstalkd.service]: 5029159663

File: adm/systemd/beanstalkd.service [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
+WantedBy=multi-user.target
```
Comment (tlstid) [inline adm/systemd/beanstalkd.service]: 5029159663

File: adm/systemd/beanstalkd.service (hunk 1 above)
Comment (pprindeville) [inline adm/systemd/beanstalkd.service]: Why does the path need to change? And why not used the $(PREFIX) in the Makefile, and add a `sed` command?

I.e. you could change this line to:

```
ExecStart=@PREFIX@/beanstalkd
```

and then have in the Makefile:

```
sed -i -e 's/@PREFIX@/$(PREFIX)/g' ...
```

when the file gets installed.

**Comment by jney** [comment]
i made this PR while i was on archlinux. which explain the path.
i don't use beanstalkd anymore, so i'm not relevant about systemd support. my opinion is, this PR should be closed.

File: adm/systemd/beanstalkd.service (hunk 3 above)
Comment (pprindeville) [reply adm/systemd/beanstalkd.service]: Eh?  Is that a phone number?

**Comment by JensRantil** [comment]
Hello! #393 proposed we move systems stuff out of this repository. Please chime in if you have any opinion.

**Comment by ysmolsky** [comment]
To be closed.

**Comment by ysmolsky** [comment]
Closing as the initial author has rejected this PR.

**Pull request closed without merging**